homeworks/szabolcsgesztesi/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,18 @@
+
+name = input("Kérlek add meg a neved: ")
+name = name.strip().capitalize()
+age = input("Kérlek add meg az életkorod: ")
+age = int(age)
+agedays = age*365
+xp = input("Kérlek add meg hány éves Python tapasztalatod van: ")
+
+#-----szorgalmi
+proxp = input("Szeretnéd, hogy a karaktered profi Python fejlesztő legyen? (yes or no) ")
+proxp = True if proxp == "yes" else False
+proxp = bool(proxp)
+
+if proxp == True:
```

Comment:
Mi lenne ha a ternary operatorod felül nem egy True-t vagy False-t adna vissza, hanem a két lenti print közötti különbséget? Ha megnézed, ugyanazt printeled ki, 1 apróságot kivéve:
"wants" vs "does not want". 
Ha a ternary kifejezés ezt a két string-et adná vissza, és ezt csak simán beágyaznád a printelt szövegbe a megfelelő helyre akkor:
-elég lenne 1 print
-nem kéne még egy if-else (itt két if-et írtál, de a lényeg ugyanaz)